setup.py

```diff
@@ -20,7 +20,15 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-from setuptools import setup
+from setuptools import find_packages,setup
 
 
-setup()
+setup(
+    package_dir= {"": "src"},
+    packages= find_packages("src"),
+    entry_points={
```

Comment:
Unsure we want to use setuptools and not the pyproject.toml for this, cc @NathanHB

Reply:
This can be done using the pyproject.toml if that's what you need, I just didn't want to touch all the configuration in pyproject.toml